lib/src/main/java/dev/openfeature/javasdk/BaseEvaluation.java

```diff
@@ -1,7 +1,7 @@
 package dev.openfeature.javasdk;
 
 /**
- * We differ between the evaluation results that providers return and what is given to the end users. This is a common interface between them.
+ * This is the common interface between the evaluation results that providers return and what is given to the end users.
```

Comment:
I try to avoid using "I/we" in favor of passive voice in comments, but this is a nitpick.